legislation_analysis/api/congress.py

```diff
@@ -161,24 +181,62 @@ def extract_text(text_url: str) -> str:
 
         return soup.text
 
+    def extract_text(self) -> None:
+        """
+        Extracts either html or pdf text for each legislation.
+        """
+        # extract html text
+        self.processed_df.loc[
+            (self.processed_df.loc[:, "text_url"].str[-3:] == "htm"),
+            "text",
+        ] = self.processed_df.loc[
+            (self.processed_df.loc[:, "text_url"].str[-3:] == "htm"),
+            "text_url",
+        ].apply(
+            lambda x: self.extract_html_text(x)
```

Comment:
Since you aren't doing anything to the value before you send it into the `extract_html_text` function, I THINK you can just do `.apply(self. extract_html_text)` without the `lambda`.